packages/server/database/migrations/20221103134908-adjustUserTiers.ts

```diff
@@ -0,0 +1,101 @@
+import {R, RDatum, RValue} from 'rethinkdb-ts'
+import getPg, {closePg} from '../../postgres/getPg'
+import IUser from '../../postgres/types/IUser'
+import segmentIo from '../../utils/segmentIo'
```

Comment:
-1 seems this is the culprit. We need to be careful with migrations and ideally avoid referring to application logic types most of the time. The reason is that these are going to change while the migration should be a stable snapshot in time.
Your particular issue is that you're indirectly referencing `getUsersByIds` which is a generated SQL query. For generating the query, the generator needs to know the database schema which it queries directly from the database. The database is only ready after all the migrations were run, thus migrations cannot refer to such generated files.

Reply:
I didn't follow it either, it was in the error message
```
requireStack: [
    '/home/runner/work/parabol/parabol/packages/server/postgres/queries/getUsersByIds.ts',
    '/home/runner/work/parabol/parabol/packages/server/utils/segmentIo.ts',
    '/home/runner/work/parabol/parabol/packages/server/database/migrations/202211031348-adjustUserTiers.ts',
    '/home/runner/work/parabol/parabol/node_modules/rethinkdb-ts-migrate/lib/migrate.js',
    '/home/runner/work/parabol/parabol/scripts/migrate.js'
  ]
```